### PR TITLE
make strndup obligatory

### PR DESCRIPTION
**Pull request by guijan**
scrot is an old program but strndup() is standardized by POSIX 2008 and widely adopted. It's time to make it obligatory.

### CODE REVIEW

File: configure.ac
Comment (daltomi) [inline configure.ac:38]: Please add the function to the required macro.
```bash
configure.ac: warning: missing AC_CHECK_FUNCS([strndup]) wanted by: src/options.c:431
```

**Review by daltomi — CHANGES_REQUESTED** [review]

**Review by daltomi — APPROVED** [review]

**Comment by daltomi** [comment]
Thanks @guijan 

Currently the project is in the stage of launching a new `release`(the `master` branch is "busy") so I leave the maintainer @eribertomota  if he wants to merge this PR at the last minute.

Cheers.

**Comment by guijan** [comment]
uh-oh. I have an important patch that will cause a merge conflict with this one, let's hold off on it for a while then.

**Comment by guijan** [comment]
Seems a merge conflict is unavoidable now because files involved with this patch were touched for the release. I'm reopening it.

**Comment by daltomi** [comment]
Ok, let me fix the conflicts via the GitHub web editor.
 I have seen that you forced the commit but it is not necessary.

**Comment by guijan** [comment]
Double check Christopher R. Nelson's copyright isn't removed. 
I actually have a properly fixed version ready to push but I don't want to cause a mess with both of us touching this PR.

**Comment by guijan** [comment]
Yes, it looks perfect now.

**Pull request merged**
